routers/user/auth.go

```diff
@@ -1035,7 +1035,7 @@ func Activate(ctx *context.Context) {
 
 		ctx.Session.Set("uid", user.ID)
 		ctx.Session.Set("uname", user.Name)
-		ctx.Redirect(setting.AppSubURL + "/")
+		ctx.Redirect(setting.AppSubURL + "/user/login")
```

Comment:
This change does not seem related to this PR

Reply:
It somehow sneaked in from #4447